app/src/main/java/io/gnosis/safe/ui/settings/app/ScreenshotAuthorizer.kt

```diff
@@ -0,0 +1,23 @@
+package io.gnosis.safe.ui.settings.app
+
+import pm.gnosis.svalinn.common.PreferencesManager
+import pm.gnosis.svalinn.common.utils.edit
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class ScreenshotAuthorizer @Inject constructor(
+    private val preferencesManager: PreferencesManager
+) {
+    fun setAllowScreenshots(value: Boolean) {
+        preferencesManager.prefs.edit {
+            putBoolean(ALLOW_SCREENSHOTS, value)
+        }
+    }
+
+    fun getAllowScreenshots(): Boolean = preferencesManager.prefs.getBoolean(ALLOW_SCREENSHOTS, false)
+
+    companion object {
+        const val ALLOW_SCREENSHOTS = "prefs.boolean.allow_screenshots"
```

Comment:
this key should be private. Also, you might add KEY_ prefix to it for clarity

Reply:
It is used in the test. Is it really a problem? I cannot be changed.